src/networks.json

```diff
@@ -801,6 +801,23 @@
     "start": 74,
     "logo": "ipfs://QmNc7QZFpPDue3Ef4SsuX55RHkqXxUxSyTCpoASeg2hW6d"
   },
+  "314": {
+    "key": "314",
+    "name": "Filecoin Mainnet",
+    "shortName": "FIL",
+    "chainId": 314,
+    "network": "mainnet",
+    "multicall": "0xAB7a22701AB6e9F7a0a295a8D8a0fBDA2C4BefAc",
```

Comment:
Please verify and publish contract code here https://filfox.info/en/address/0xAB7a22701AB6e9F7a0a295a8D8a0fBDA2C4BefAc